tests/ui/const-generics/generic_const_exprs/unify-op-with-fn-call.rs

```diff
@@ -1,36 +0,0 @@
-//@ known-bug: #110395
-//@ compile-flags: -Znext-solver
-#![feature(generic_const_exprs, adt_const_params, const_trait_impl)]
```

Comment:
`known-bug` `-Z-next-solver` `feature(generic_const_expr)` tests seem a little pointless seeing as how these features are known to not work together and I do not think there is any intention of making them work together.